examples/playwright/src/tests/theia-preference-view.test.ts

```diff
@@ -38,44 +38,63 @@ test.describe('Preference View', () => {
         expect(await preferences.getStringPreferenceById(preferenceId)).toBe('8000');
 
-        await preferences.resetStringPreferenceById(preferenceId);
+        await preferences.resetPreferenceById(preferenceId);
         expect(await preferences.getStringPreferenceById(preferenceId)).toBe(DefaultPreferences.DiffEditor.MaxComputationTime);
     });
 
     test('should be able to read, set, and reset Boolean preferences', async () => {
         const preferences = await app.openPreferences(TheiaPreferenceView);
         const preferenceId = PreferenceIds.Explorer.AutoReveal;
 
-        await preferences.resetBooleanPreferenceById(preferenceId);
+        await preferences.resetPreferenceById(preferenceId);
         expect(await preferences.getBooleanPreferenceById(preferenceId)).toBe(DefaultPreferences.Explorer.AutoReveal.Enabled);
 
         await preferences.setBooleanPreferenceById(preferenceId, false);
         await preferences.waitForModified(preferenceId);
         expect(await preferences.getBooleanPreferenceById(preferenceId)).toBe(false);
 
-        await preferences.resetBooleanPreferenceById(preferenceId);
+        await preferences.resetPreferenceById(preferenceId);
         expect(await preferences.getBooleanPreferenceById(preferenceId)).toBe(DefaultPreferences.Explorer.AutoReveal.Enabled);
     });
 
+    test('should be able to read, set, and reset Options preferences', async () => {
+        const preferences = await app.openPreferences(TheiaPreferenceView);
+        const preferenceId = PreferenceIds.Editor.RenderWhitespace;
+
+        await preferences.resetPreferenceById(preferenceId);
+        expect(await preferences.getOptionsPreferenceById(preferenceId)).toBe(DefaultPreferences.Editor.RenderWhitespace.Selection);
+
+        await preferences.setOptionsPreferenceById(preferenceId, DefaultPreferences.Editor.RenderWhitespace.Boundary);
+        await preferences.waitForModified(preferenceId);
+        expect(await preferences.getOptionsPreferenceById(preferenceId)).toBe(DefaultPreferences.Editor.RenderWhitespace.Boundary);
+
+        await preferences.resetPreferenceById(preferenceId);
+        expect(await preferences.getOptionsPreferenceById(preferenceId)).toBe(DefaultPreferences.Editor.RenderWhitespace.Selection);
+    });
+
     test('should throw an error if we try to read, set, or reset a non-existing preference', async () => {
         const preferences = await app.openPreferences(TheiaPreferenceView);
 
         preferences.customTimeout = 500;
         try {
             await expect(preferences.getBooleanPreferenceById('no.a.real.preference')).rejects.toThrowError();
             await expect(preferences.setBooleanPreferenceById('no.a.real.preference', true)).rejects.toThrowError();
-            await expect(preferences.resetBooleanPreferenceById('no.a.real.preference')).rejects.toThrowError();
+            await expect(preferences.resetPreferenceById('no.a.real.preference')).rejects.toThrowError();
```

Comment:
minor: we should probably update the preference in the future but it's not critical:

```suggestion
            await expect(preferences.resetPreferenceById('not.a.real.preference')).rejects.toThrowError();
```

Reply:
Nice catch, thanks! I fixed this typo in fb61ad3